kotlin/PixelPass/src/main/java/io/mosip/pixelpass/PixelPass.kt

```diff
@@ -86,14 +86,22 @@ class PixelPass {
 
         val payload = Utils().toDataItem(mappedJson)
 
-        val baos = ByteArrayOutputStream()
-        CborEncoder(baos).encode(payload)
-        return baos.toByteArray().toHexString()
+        if (cborEnable) {
+            val baos = ByteArrayOutputStream()
```

Comment:
can we use better variable name for `baos`

Reply:
Done